app/models/invenio_rdm_record_converter.rb

```diff
@@ -611,4 +611,14 @@ def publisher
   def pediatric_neurology_brief?
     @generic_file.depositor == INSTITUTIONAL_PNB_DEPOSITOR || @generic_file.doi.shift.include?(PNB_DOI_PREFIX)
   end
+
+  def sizes
+    if @generic_file.id == "3105875f-61e1-412c-9b1c-c8a33b37ff35"
```

Comment:
This seems like the fastest and simplest solution. If we find more situations like this we can just add more elif statements.